client/reader/sidebar/index.jsx

```diff
@@ -53,6 +55,28 @@ import ReaderSidebarTags from './reader-sidebar-tags';
 import 'calypso/my-sites/sidebar/style.scss'; // Copy styles from the My Sites sidebar.
 import './style.scss';
 
+//TODO: Remove this component once the new reader MSD is enabled for all users
+const DeprecatedReaderSidebarHeader = ( { onSearchClicked } ) => {
```

Comment:
We shouldn't hide the new location of the search behind the feature flag. It will make things more simple.